### ISSUE TITLE
New test runner: Use npm test command

### ISSUE THREAD

**Issue by nicojs** [stryker-mutator/stryker#768]
We can simplify the use of Stryker quite a bit if we simply use the `npm test` command of a repository. In order to do this, we can create a simple test runner, let's call it the `NpmTestTestRunner` for now. 

The idea is that, instead of hooking into the test runner and test framework, we simply use the `npm test` command. If exit code is 0, the test passed, if it is 1, the test errors. Our timeout system should still work with this approach, so a timeout would also be supported.

Basically, we would trade in performance for simplicity.

[Mutode](https://github.com/TheSoftwareDesignLab/mutode) uses this approach. @DiegoRBaquero is the author and might have some time to implement this feature.

**Comment by simondel** [comment]
That's quite a clean way to incorporate it in Stryker :) do you want this in the core Stryker package or as a plugin? This may have zero dependencies.

**Comment by nicojs** [comment]
Good question. I'm happy with putting it in the core, as long as it has 0 dependencies and it is not MB's in size (which is shouldn't have to be). 

We might want to make it more generic in that we might want to allow people to inject there own test script. For example `mocha test/**/*.js`. That might come in handy, if people also use e2e tests as part of there normal build process and want Stryker to just run the unit tests. 

So the name could better be `CustomScriptTestRunner`, or `CustomCommandTestRunner`.

**Comment by nicojs** [comment]
I had some time on a 10 hour flight, so I decided to implement this feature. @DiegoRBaquero do you want to review it? I'm especially interested to see if made some obvious mistakes using `require('child_process').exec`.

**Comment by DiegoRBaquero** [comment]
Cool! I'll try to review it this weekend
